cloudbuild.yml

```diff
@@ -0,0 +1,129 @@
+steps:
+# Debian 8
+  - name: 'gcr.io/cloud-builders/docker'
+    args: 
+      - build
+      - -f
+      - bb-worker/Dockerfile.debian8-build-python3.7
+      - --tag=debian8-build-python:3.7
+      - bb-worker/
+    id: debian8-build-python
+    wait_for: ['-']
+  - name: 'gcr.io/cloud-builders/docker'
+    args: 
+      - build
+      - -f
+      - bb-worker/Dockerfile.debian8.python3.7
+      - --tag=gcr.io/zcash-web/bb-worker-debian8:python3.7
+      - bb-worker/
+    wait_for: ['debian8-build-python']
+
+# Debian 9
+  - name: 'gcr.io/cloud-builders/docker'
+    args: 
+      - build
+      - -f
+      - bb-worker/Dockerfile.apt-build-python3.7
+      - --tag=debian9-build-python:3.7
+      - --build-arg=BASEOS=debian:9
+      - bb-worker/
+    id: debian9-build-python
+    wait_for: ['-']
+  - name: 'gcr.io/cloud-builders/docker'
+    args: 
+      - build
+      - -f
+      - bb-worker/Dockerfile.apt.python3.7
+      - --tag=gcr.io/zcash-web/bb-worker-debian9:python3.7
+      - --build-arg=FROMBASEOS=debian9
+      - --build-arg=BASEOS=debian9
+      - bb-worker/
+    wait_for: ['debian9-build-python']
+
+# Debian 10
+  - name: 'gcr.io/cloud-builders/docker'
+    args: 
+      - build
+      - -f
+      - bb-worker/Dockerfile.apt-build-python3.7
+      - --tag=debian10-build-python:3.7
+      - --build-arg=BASEOS=debian:10
+      - bb-worker/
+    id: debian10-build-python
+    wait_for: ['-']
+  - name: 'gcr.io/cloud-builders/docker'
+    args: 
+      - build
+      - -f
+      - bb-worker/Dockerfile.apt.python3.7
+      - --tag=gcr.io/zcash-web/bb-worker-debian10:python3.7
+      - --build-arg=FROMBASEOS=debian10
+      - --build-arg=BASEOS=debian10
+      - bb-worker/
+    wait_for: ['debian9-build-python']
+
+# Ubuntu 16.04
+  - name: 'gcr.io/cloud-builders/docker'
+    args: 
+      - build
+      - -f
+      - bb-worker/Dockerfile.apt-build-python3.7
+      - --tag=ubuntu1604-build-python:3.7
+      - --build-arg=BASEOS=ubuntu:16.04
+      - bb-worker/
+    id: ubuntu1604-build-python
+    wait_for: ['-']
+  - name: 'gcr.io/cloud-builders/docker'
+    args: 
+      - build
+      - -f
+      - bb-worker/Dockerfile.apt.python3.7
+      - --tag=gcr.io/zcash-web/bb-worker-ubuntu1604:python3.7
+      - --build-arg=FROMBASEOS=ubuntu1604
+      - --build-arg=BASEOS=ubuntu1604
+      - bb-worker/
+    wait_for: ['debian9-build-python']
```

Comment:
should this be `ubuntu1604-build-python`?

Reply:
Yes, updated!